kind-1.19.yaml

```diff
@@ -3,3 +3,16 @@ apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
   image: kindest/node:v1.19.16@sha256:476cb3269232888437b61deca013832fee41f9f074f9bed79f57e4280f7c48b7
+  kubeadmConfigPatches:
+    - |
+      kind: InitConfiguration
+      nodeRegistration:
+       kubeletExtraArgs:
+         node-labels: "ingress-ready=true"
+  extraPortMappings:
```

Comment:
What're these for?

Reply:
it is required for ingress

Reply:
Would it still be able to run multiple kind clusters with this change at the same time?

Reply:
it probably opens a port on the hosts, so no